src/web/GestureView.tsx

```diff
@@ -182,6 +183,27 @@ export class GestureView extends React.Component<Types.GestureViewProps, Types.S
         }
     }
 
+    private _sendContextMenuEvent = (e: React.MouseEvent<any>) => {
+        if (this.props.onContextMenu) {
+            e.preventDefault();
+            e.stopPropagation();
```

Comment:
I moved these a bit earlier to keep this behavior on par with the rest on reactxp - events are being cancelled before their synthentic handlers. At the same time if onContextMenu is not defined, I don't think we should prevent the event from bubbling down.